lib/pbench/server/filetree.py

```diff
@@ -1,12 +1,13 @@
+import re
+import shutil
+import subprocess
+import tarfile
+
 from configparser import ConfigParser
 from logging import Logger
 from pathlib import Path
-import re
-import shutil
 from typing import Dict, Optional, Union
```

Comment:
I always thought we are doing this.

Reply:
We should probably discuss this briefly in SCRUM, just so that we're all on the same page.

[PEP-8 outlines the convention](https://peps.python.org/pep-0008/#imports) of the three blocks; however, it doesn't say what to do within the blocks.  Personally, I like having the `from`'s separated from the plain `import`'s, but doing so has the downside of making it less obvious when a module has _both_ a `from` and a plain `import` targeting the same module (which should be avoided, unless the import target is being renamed, which is atypical), so it's probably best to stick to just three blocks, with `from`'s and plain `import`'s intermingled and sorted by module name.

Reply:
I've generally tried to keep them sorted by module name rather than by `import`/`from`, and I think that's a result of some previous discussions within the team but possibly not a conscious team decision. 👀

Reply:
As per the Discussion with the team. We will be moving forward with having a sorted single Python Standard Library Packages block.